state-synchronizer/src/executor_proxy.rs

```diff
@@ -86,16 +86,16 @@ impl ExecutorProxyTrait for ExecutorProxy {
     ) -> Pin<Box<dyn Future<Output = Result<SynchronizerState>> + Send>> {
         let client = Arc::clone(&self.storage_read_client);
         async move {
-            let highest_local_li = client.update_to_latest_ledger_async(0, vec![]).await?.1;
-            // highest committed version is max between LI and synced tree state
-            let mut highest_committed_version = highest_local_li.ledger_info().version();
-            if let Some(startup_info) = client.get_startup_info_async().await? {
-                let pending_version = startup_info.synced_tree_state.map_or(0, |t| t.version);
-                highest_committed_version =
-                    std::cmp::max(pending_version, highest_committed_version);
-            }
+            let storage_info = client
+                .get_startup_info_async()
+                .await?
+                .ok_or_else(|| format_err!("[state sync] Failed to access storage info"))?;
+            let highest_committed_version = std::cmp::max(
```

Comment:
Not related. But IIUC, the `committed` naming here is confusing. Should it be called `synced`